### ISSUE TITLE
CMD + K is not working when the focus is on compose box

### ISSUE THREAD

**Issue by isagoico** [Expensify/Expensify.cash#3512]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Expected Result:
User should be able to change chats when using the command + k. 

## Actual Result:
CMD + K is not working when the focus is on compose box.

## Action Performed:
1. Log in to e.cash 
2. Navigate to a conversation
3. Set the focus to the compose box
4. Hit CMD + K

## Workaround:
User has to manually set the focus on the chat switcher

## Platform:
Where is this issue occurring?

 Web ✔️ 
 iOS
 Android
 Desktop App
 Mobile Web

**Version Number:**1.0.66-0

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** 

![image](https://user-images.githubusercontent.com/44479856/121524609-c1c5c180-c9f7-11eb-93ef-3d87f98fb8c0.png)

**Expensify/Expensify Issue URL:**

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

___ 
From @iwiznia https://expensify.slack.com/archives/C01GTK53T8Q/p1623271189154800

> CMD+K shortcut is not working when the focus is in the chat text box (tested on web in staging, if this is not happening in prod, we should block deploy on it) 

Additional info:
This change was caused by upgrading our react-native-web package to 0.15.7. 
It appears that when focused on inputs, all existing keybinds no longer work.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @stitesExpensify (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by stitesExpensify** [comment]
Checked out the commit before bumping the [react-native-web version](https://github.com/Expensify/Expensify.cash/pull/3215) and this bug no longer exists. I'm not sure why that is though

**Comment by marcaaron** [comment]
I am going to need to have someone else look into this for me for `#focus`. I'd suggest we maybe create a new issue and move forward with the deploy.

**Comment by stitesExpensify** [comment]
Removing deploy blocker label and adding external

**Comment by MelvinBot** [comment]
Triggered auto assignment to @arielgreen (`External`), see https://stackoverflow.com/c/expensify/questions/8582 for more details.

**Comment by arielgreen** [comment]
I'm a little unclear, should I still get this up on Upwork?

**Comment by iwiznia** [comment]
I don't know. IMO either we should revert the PR that broke this or fix it. @stitesExpensify why do you say we can't revert the PR that broke this?

**Comment by roryabraham** [comment]
I think I agree with @stitesExpensify on this one – we probably don't want to revert https://github.com/Expensify/Expensify.cash/pull/3215. We won't be doing a prod deploy today anyways, so we have tomorrow to try and get this fixed, so that would be ideal. But otherwise I think we should just :shipit: and not treat this as a deploy blocker.

**Comment by trjExpensify** [comment]
DB's stay internal because they are `hourly` priority (unless that's changed), so I'm removing the label for the time being.

I agree with Rory, let's use tomorrow to assess what the fix is for this. If it's straightforward, let's do it and avoid shipping a regression that has a pretty big impact on everyone's productivity in E.cash. If we end up needing more time, then I think we'll need to remove it as a deploy blocker, so that we can get a new prod build to Apple and unblock N5.

**Comment by Julesssss** [comment]
I also don't think this should be a deployBlocker. But I will take a look at the issue today to see if there's a simple fix.

**Comment by Julesssss** [comment]
No luck. I [added some notes](https://github.com/Expensify/Expensify.cash/pull/3215#issuecomment-859564330) on the PR which introduced this library change. Moving onto other deploy blockers.

**Comment by trjExpensify** [comment]
👋 @HorusGoul found the fix in [this thread](https://expensify.slack.com/archives/C03TQ48KC/p1623429811172500?thread_ts=1623429012.170300&cid=C03TQ48KC). Over to you, @roryabraham!

**Comment by roryabraham** [comment]
Thanks to @HorusGoul the PR to fix this is [ready for review](https://github.com/Expensify/Expensify.cash/pull/3555)
